packages/server-reason-react-ppx/cram/styles.t/run.t

```diff
@@ -8,3 +8,128 @@ We need to output ML syntax here, otherwise refmt could not parse it.
          Some (React.JSX.Style (snd x : ReactDOM.Style.t));
        ])
     []
+  ;;
+  
+  React.createElement "div"
+    (Stdlib.List.filter_map Stdlib.Fun.id
+       [
+         (match
+            (match x with None -> None | Some x -> Some (fst x) : string option)
+          with
+         | None -> None
+         | Some v -> Some (React.JSX.String ("class", "className", v)));
+         (match
+            (match x with None -> None | Some x -> Some (snd x)
+              : ReactDOM.Style.t option)
+          with
+         | None -> None
+         | Some v -> Some (React.JSX.Style v));
```

Comment:
this looks like unnesesary match expressions, did you check the output? Why make it like this?